fastparquet/test/test_output.py

```diff
@@ -1,3 +1,5 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
```

Comment:
Fair enough.  The problem was unicode literals in some of the assertions.  Setting these explicitly to unicode seems to solve the problem.